src/utils/importFuncs.ts

```diff
@@ -0,0 +1,163 @@
+function cairoCommonPath(path: string[]): string[] {
+  return ['starkware', 'cairo', 'common', ...path];
+}
```

Comment:
Let's have it as a const, and then we'd do:  `const ALLOC = [[...CAIRO_COMMON_PATH, 'alloc], 'alloc]`

Reply:
At first I thought not to use constants in order to not cause reference issues. But you are right, these should be changed to variables instead.